main_test.go

```diff
@@ -134,8 +134,41 @@ func TestStart(t *testing.T) {
 		exitStatus <- status
 	})
 
+	t.Run("it sends SIGTERM with signalForwardingDelay", func(t *testing.T) {
+		signalForwardingDelay := uint(2)
+		cp, err := start([]string{"sleep", "10"}, []string{"linked"}, timeout, signalForwardingDelay)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		exitStatus := make(chan int)
+		go func() {
+			process, _ := os.FindProcess(os.Getpid())
+			process.Signal(syscall.SIGTERM)
+			linkedContainer.KnownStatus = "STOPPED"
+
+			want := 128 + int(syscall.SIGTERM)
+			select {
+			case got := <-exitStatus:
+				if got != want {
+					t.Errorf("exitStatus: got: %v, want: %v", got, want)
+				}
+				return
+			case <-time.After(time.Duration(timeout+signalForwardingDelay+1) * time.Second):
+				t.Errorf("timeout")
+			}
+			close(exitStatus)
```

Comment:
I forgot to close the channel in other tests 💦 
By the way, I prefer to use `defer` because this line sometimes won't be executed.

Reply:
fixed https://github.com/abicky/ecswrap/pull/1/commits/c2044308b969e4276ac298620bb3b7d627f13fe7